rebasehelper/helpers/rpm_helper.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os
+import re
+import tempfile
+
+import rpm
+import six
+
+from rebasehelper.constants import defenc
+from rebasehelper.exceptions import RebaseHelperError
+from rebasehelper.logger import logger
+from rebasehelper.helpers.macro_helper import MacroHelper
+from rebasehelper.helpers.process_helper import ProcessHelper
+from rebasehelper.helpers.console_helper import ConsoleHelper
+
+
+class RpmHelper(object):
+
+    """Class for working with RPM databases and packages."""
+
+    ARCHES = None
+
+    @staticmethod
+    def is_package_installed(pkg_name=None):
+        """Checks whether a package is installed.
+
+        Args:
+            pkg_name (str): Name of the package.
+
+        Returns:
+            bool: Whether the package is installed.
+
+        """
+        ts = rpm.TransactionSet()
+        mi = ts.dbMatch('provides', pkg_name)
+        return len(mi) > 0
+
+    @staticmethod
+    def all_packages_installed(pkg_names=None):
+        """Check if several packages are installed.
+
+        Args:
+            pkg_names (list): List of package names to check.
+
+        Returns:
+            bool: True if all packages are installed, False otherwise.
+
+        """
+        for pkg in pkg_names:
+            if not RpmHelper.is_package_installed(pkg):
+                return False
+        return True
+
+    @staticmethod
+    def install_build_dependencies(spec_path=None, assume_yes=False):
+        """Install all build requirements of a package using PolicyKits.
+
+        Args:
+            spec_path (str): Absolute path to the SPEC file.
+            assume_yes (bool): Whether to use -y parameter for the installation.
```

Comment:
This is not very useful, rather: `Whether to automatically answer yes to all questions.`.